core/src/edu/cornell/gdiac/temporary/Level.java

```diff
@@ -203,6 +210,15 @@ public void setsThreshold(long sThreshold) {
     JsonValue data;
 
     float maxSample;
+
+    // this comparator is used for comparing comp flag data later on
+    private static class CustomComparator implements Comparator<Long[]> {
+        @Override
+        public int compare(Long[] o1, Long[] o2) {
+            return Long.compare(o1[0], o2[1]);
+        }
+    }
+
```

Comment:
for the sake of performance issues, don't use a sort here (I'm assuming that you are sorting the stuff in the level when it loads in)

I think we need to have the level editor make the sort.